source/guide/3.2/plugin-softlayer.md

```diff
@@ -0,0 +1,499 @@
+    1. Validates properties as done in creation_validation
+    2. Creates a virtual server.
+    3. Waits for transactions to begin (the create process has begun).
+  * `cloudify.interfaces.lifecycle.start`
+    1. Waits for transactions to end if there are any.
+    2. Starts the server, if it’s not already running.
+  * `cloudify.interfaces.lifecycle.stop`
+    1. Waits for transactions to end if there are any.
+    2. Stops the server, if it’s not already halted.
+  * `cloudify.interfaces.lifecycle.delete`
+    1. Deletes the server.
+    2. Waits for transactions to start (the delete process has begun). 
+    3. Waits for for transactions to end (the delete process has terminated).
+  * `cloudify.interfaces.validation.creation_validation`
+    1. Validates that all required properties are specified.
+    2. Validates that exactly one of the properties `OS`, `image_template_global_id` or `image_template_id` were provided.
+    3. Validates that all specified price ids are legal.
+    4. Validates private only properties.
+
+
+**Attributes:**
+  See the [Runtime Properties section](#runtime-properties)
+
+
+## cloudify.softlayer.nodes.WindowsServer
+
+
+**Derived From:** [cloudify.softlayer.nodes.VirtualServer](#cloudifysoftlayernodesvirtualserver)
+	This type has the same properties and operations-mapping as the type above (as it is derived from it), 
+	yet it overrides some of the agent and plugin installations operations-mapping derived from the built-in cloudify.nodes.Compute type. 
+	
+  Use this type when working with a Windows server.
+
+
+# Runtime Properties
+
+Node instances of any of the types defined in this plugin are set with the following runtime properties:
+
+Two runtime-properties are available on node instances of these types once the `cloudify.interfaces.lifecycle.create` operation succeeds:
+  
+  * `instance_id` – virtual server instance unique identifier 
+  * `hostname` – the hostname determined in the plugin's creation process.
+
+The following runtime-properties are available on node instances of this type once the `cloudify.interfaces.lifecycle.start` operation succeeds: 
+  
+  * `ip` – the private ip of the server
+  * `public_ip` – the public ip of the server
+  * `username` – server's username
+  * `password` – server's password
+
+
+# Resource Naming Convention
+When creating a virtual server, its name on SoftLayer will be `<hostname>.<domain>` where \<hostname> and \<domain> are the values of the `hostname` and the `domain` proeprties respectively.
+
+In case the `hostname` property is not provided, the value of the server's hostname will default to its node-instance-id, which was generated in the deployment creation process. 
+
+{%note%}
+In case this server is created as part of a deployment on a manager, and the `resource_prefix` property is provided on the manager blueprint, the server's full name on SoftLayer will be `<prefix>-<hostname/node-instance-id>.<domain>`.
```

Comment:
The syntax here is wrong. Not sure what is meant here. Maybe it should be like this:
In case this server is created as part of a deployment that is running on a manager, and the `resource_prefix` property is provided in the manager blueprint, the server's full name on SoftLayer will be `<prefix>-<hostname/node-instance-id>.<domain>`.